variables/typography.scss

```diff
@@ -54,15 +54,15 @@ $typography: (
   ),
   p: (
     family: $font-family,
-    size: .875rem,
+    size: 1rem,
```

Comment:
Endrer du ikke disse i den andre pr'n din?

Reply:
Jo, var for å sjekke størrelse, men den kom med her nå. Tar å fjerner foreløpig